source/_integrations/hue.markdown

```diff
@@ -12,73 +12,48 @@ ha_quality_scale: platinum
 ha_codeowners:
   - '@balloob'
   - '@frenck'
+  - '@marcelveldt'
 ha_domain: hue
 ha_ssdp: true
 ha_homekit: true
 ha_platforms:
   - binary_sensor
   - light
   - sensor
+  - scene
 ha_zeroconf: true
 ---
 
-The Philips Hue integration allows you to control and monitor the lights and motion sensors connected to your Hue bridge.
+The Philips Hue integration allows you to control and monitor the lights and sensors connected to your Hue bridge.
 
 There is currently support for the following device types within Home Assistant:
 
 - Lights
 - Motion sensors (including temperature and light level sensors)
-- Hue switches (as device triggers for automations and also exposed as battery sensors when they are battery-powered)
-  - Hue Dimmer Switch
-  - Hue Tap Switch
-  - Hue Smart Button
-  - Hue Wall Switch Module
-  - Friends of Hue Switch
+- Hue remotes/switches (as device triggers for automations and also exposed as battery sensors when they are battery-powered)
 
 {% include integrations/config_flow.md %}
 
-## Options
+## Lights for Hue zones and rooms
 
-Options for the Hue integration can be set going to **Configuration** -> **Integrations** -> **Hue** -> **Options**.
+The Hue concept is based on Rooms and zones. Altough the underlying Hue lights are exposed directly to Home Assistant it might also be useful to interact with the `grouped lights` of the Hue ecosystem, for example to turn all lights in a Hue group on/off at the same time.
```

Comment:
```suggestion
The Hue concept is based on Rooms and zones. Although the underlying Hue lights are exposed directly to Home Assistant it might also be useful to interact with the `grouped lights` of the Hue ecosystem, for example to turn all lights in a Hue group on/off at the same time.
```